### DISCUSSION TITLE
Putting device tracker to unknown while watch is charging

### DISCUSSION THREAD

**Discussion by JoergBr** [Ludy87/xplora_watch#285] [Q&A]
When my daughter puts her Xplora on the charger, of course she is not wearing it. Still she (the home assistant person with the Xplora device tracker) is showing up as "home" which is a problem for other automations when she actually is not home. Would it be possible to change the location to "Unknown" while the watch is charging?

**Comment by Ludy87** [comment]
You need to integrate a condition into the automations in Home Assistant.

**Comment by JoergBr** [comment]
But in which automations? The xplora tracker is directly connected to the person, so at least there's no self-created automation so far. And I see no possibility to change the tracker sensor to "unknown" manually.
________________________________
Von: Ludy ***@***.***>
Gesendet: Donnerstag, 12. Oktober 2023 10:18
An: Ludy87/xplora_watch ***@***.***>
Cc: JoergBr ***@***.***>; Author ***@***.***>
Betreff: Re: [Ludy87/xplora_watch] Putting device tracker to unknown while watch is charging (Discussion #285)


You need to integrate a condition into the automations in Home Assistant.

**Comment by Ludy87** [comment]
I don't fully understand the problem, the watch is at home in the charging station, then it is physically at home.

**Comment by JoergBr** [comment]
Of course, but in the end it is our daughter we want to track and not the watch. But if there's no easy solution or the solution lies outside the xplora add-in, never mind.
Thank you for the great add-in which makes the usage possible in the first place!

________________________________
Von: Ludy ***@***.***>
Gesendet: Freitag, 13. Oktober 2023 09:15
An: Ludy87/xplora_watch ***@***.***>
Cc: JoergBr ***@***.***>; Author ***@***.***>
Betreff: Re: [Ludy87/xplora_watch] Putting device tracker to unknown while watch is charging (Discussion #285)


I don't fully understand the problem, the watch is at home in the charging station, then it is physically at home.
